Static/Section.swift

```diff
@@ -13,6 +13,9 @@ public struct Section: Hashable, Equatable {
         /// Custom view for the header or footer. The height will be the view's `bounds.height`.
         case view(UIView)
 
+        // View Sized with autolayout
+        case autoLayoutView(UIView)
```

Comment:
If using this view, must set estimated header/footer height to > 0, if using tableView's before iOS11